src/clarity-angular/popover/signpost/signpost-trigger.directive.ts

```diff
@@ -8,7 +8,10 @@ import {Subscription} from "rxjs/Subscription";
 
 import {IfOpenService} from "../../utils/conditional/if-open.service";
 
-@Directive({selector: "[clrSignpostTrigger]"})
+@Directive({
+    selector: "[clrSignpostTrigger]",
+    host: {"[class.signpost-trigger]": "true", "[attr.tabindex]": "0", "role": "button"}
```

Comment:
tab index is set to 0 always? Would that affect the focus order on the page?

Reply:
@adityarb88 `tabindex = 0`  obeys the sequential document tab order. So it should be fine. We have the same thing in our tooltips as well

Reply:
+1 to what @Shijir said

Reply:
👍